brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapErrors.swift

```diff
@@ -31,59 +31,55 @@ enum SwapErrors: FEError {
     case pendingSwap
     case selectAssets
     
+    // TODO: localize
     var errorMessage: String {
         switch self {
         case .balanceTooLow(let balance, let currency):
-            return String(format: "You don't have enough %@ to complete this swap. Your current %@ balance is %@",
-                          currency,
-                          currency,
-                          ExchangeFormatter.crypto.string(for: balance) ?? "0.00")
+            return L10n.ErrorMessages.balanceToLow(currency, currency, ExchangeFormatter.crypto.string(for: balance) ?? "0.00"))
             
         case .tooLow(let amount, let currency):
-            return String(format: "The amount is lower than the minimum of %.1f %@. Please enter a higher amount.",
-                          amount.doubleValue,
-                          currency)
+            return L10n.ErrorMessages.amountToLow(Int(amount.doubleValue), currency)
             
         case .tooHigh(let amount, let currency):
-            return String(format: "The amount is higher than the swap maximum of %.2f %@.",
-                          amount.doubleValue,
-                          currency)
+            return L10n.ErrorMessages.swapAmountToHigh(amount, currency)
             
         case .overDailyLimit:
-            return "The amount is higher than your daily limit of $1,000 USD. Please upgrade your account or enter a lower amount."
+            return L10n.ErrorMessages.overDailyLimit
             
         case .overLifetimeLimit:
-            return "The amount is higher than your lifetime limit of $10,000 USD. Please upgrade your account or enter a lower amount."
+            return L10n.ErrorMessages.overLifetimeLimit
             
         case .overDailyLimitLevel2:
-            return "The amount is higher than your daily limit of $10,000 USD. Please enter a lower amount."
+            return L10n.ErrorMessages.overLifetimeLimitLevel2
             
         case .noFees:
-            return "Failed to fetch network fees. Please try again later."
+            return L10n.ErrorMessages.noFees
             
         case .networkFee:
-            return "This swap doesn't cover the included network fee. Please add more funds to your wallet or change the amount you're swapping."
+            return L10n.ErrorMessages.networkFee
             
-        case .noQuote(let pair):
-            return "No quote for currency pair \(pair ?? "<missing>")."
+        case .noQuote(let from, let to):
+            let from = from ?? "/"
+            let to = to ?? "/|"
```

Comment:
"/"

Reply:
what would u replace an optional war with?

Reply:
just saying there is a typo. it should be "/" not "/|"